recipes/acts/recipe.yaml

```diff
@@ -0,0 +1,58 @@
+context:
+  name: acts
+  version: "41.1.0"
+
+recipe:
+  name: ${{ name }}-suite
+  version: ${{ version }}
+
+source:
+  url: https://github.com/acts-project/acts/archive/refs/tags/v${{ version }}.tar.gz
+  sha256: 2cd559003d7c37c089adef0c357b7cb8dee7d2fa55f26d83ac41b3e9a2a1c22c
+
+build:
+  number: 0
+  skip: win
+
+outputs:
+  - package:
+      name: ${{ name|lower }}-core
+
+    build:
+      script:
+        - if: osx
+          then: export CXXFLAGS="${CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY"
```

Comment:
Can you add a comment here that adds links to the relevant supporting conda-forge GitHub Issues or PRs to motivate this?